app/views/ReadReceiptView/index.js

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FlatList, View, Text } from 'react-native';
+import { SafeAreaView } from 'react-navigation';
+import equal from 'deep-equal';
+import moment from 'moment';
+import { connect } from 'react-redux';
+
+import Avatar from '../../containers/Avatar';
+import styles from './styles';
+import RCActivityIndicator from '../../containers/ActivityIndicator';
+import I18n from '../../i18n';
+import RocketChat from '../../lib/rocketchat';
+import StatusBar from '../../containers/StatusBar';
+
+@connect(state => ({
+	Message_TimeFormat: state.settings.Message_TimeFormat,
+	baseUrl: state.settings.Site_Url || state.server ? state.server.server : '',
+	userId: state.login.user && state.login.user.id,
+	token: state.login.user && state.login.user.token
+}))
+export default class ReadReceiptsView extends React.Component {
+	static navigationOptions = {
+		title: I18n.t('Read_Receipt')
+	}
+
+	static propTypes = {
+		navigation: PropTypes.object,
+		Message_TimeFormat: PropTypes.string,
+		baseUrl: PropTypes.string,
+		userId: PropTypes.string,
+		token: PropTypes.string
+	}
+
+	constructor(props) {
+		super(props);
+		this.messageId = props.navigation.getParam('messageId');
+		this.state = {
+			loading: false,
+			receipts: []
+		};
+	}
+
+	componentDidMount() {
+		this.load();
+	}
+
+	shouldComponentUpdate(nextProps, nextState) {
+		const { loading, receipts } = this.state;
+		if (nextState.loading !== loading) {
+			return true;
+		}
+		if (!equal(nextState.receipts, receipts)) {
+			return true;
+		}
+		return false;
+	}
+
+	load = async() => {
```

Comment:
It loads only 50 rows per time, right?
Shouldn't we have a `loadMore` or something?